packages/bigcommerce-payments-utils/src/bigcommerce-payments-sdk.ts

```diff
@@ -0,0 +1,249 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import {
+    MissingDataError,
+    MissingDataErrorType,
+    PaymentMethod,
+    PaymentMethodClientUnavailableError,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import {
+    BigCommercePaymentsHostWindow,
+    BigCommercePaymentsInitializationData,
+    PayPalFastlaneSdk,
+    PayPalMessagesSdk,
+    PayPalSdkConfig,
+} from './bigcommerce-payments-types';
+
+export default class BigCommercePaymentsSdk {
+    private window: BigCommercePaymentsHostWindow;
```

Comment:
It might be better to rename this class to `PayPalSdkHelper`, since it contains logic related to PayPal SDK configuration. Also `BigCommercePaymentsHostWindow` should be changed to `PayPalHostWindow`, because window object will contain paypal related objects and method after PayPal SDK is initialised.

```suggestion
export default class PayPalSdkHelper {
    private window: PayPalHostWindow;
```